cmd/axon/sql.go

```diff
@@ -132,17 +134,18 @@ func insertRow(sourceDB *sqlx.DB, targetDB *sqlx.DB, schema string, change *warp
 		// PG error codes: https://www.postgresql.org/docs/9.2/errcodes-appendix.html
 		pqe, ok := err.(*pq.Error)
 		if !ok {
-			return fmt.Errorf("failed to insert %s for query %s: %+v", change, removeDuplicateSpaces(query), err)
+			return fmt.Errorf("failed to insert %s for query %s args %s: %+v", change, removeDuplicateSpaces(query), args, err)
 		}
 		if pqe.Code.Name() == "unique_violation" {
 			// Ignore duplicates
 			// TODO: Should they be updated instead?
 			log.Printf("duplicate row insert skipped %s:", change)
 			// Always update, even on duplicate row.
-			err = updateColumnSequence(targetDB, change.Table, change.NewValues)
-			if err != nil {
-				return err
-			}
+			// DISABLED: Angel believes this is the problem. I doubt it, but trying won't hurt.
```

Comment:
If this is to stay, it should outline the "problem".

Reply:
Removed.